src/advanced/time.md

```diff
@@ -2,10 +2,9 @@
 
 <!-- cspell:ignore tlsdate,roughtime -->
 
-[**exonum-time**][exonum-time] is a time oracle service for Exonum.
-This service allows determining time,
-importing it from the external world to the blockchain
-and keeping its current value in the blockchain.
+Time Oracle allows user services to access the calendar time supplied by
+validator nodes to the blockchain. Its implementations for Exonum are available
```

Comment:
Time Oracle allows user services to access the calendar time which validator nodes supply to the blockchain from the external world.

Reply:
May I ask why do you think `the external world` part is important? In my opinion it adds unnecessary ambiguity - if I understand correctly, validator nodes supply their own local time, in this context `the external world` seems kind of misleading (like they get it from some third-party source of truth).